include/blosc2.h

```diff
@@ -37,7 +37,7 @@ extern "C" {
 #endif
 
 #if defined(_WIN32) && !defined(__MINGW32__)
-#include <windows.h>
+  #include <windows.h>
   #include <malloc.h>
```

Comment:
Perhaps it's best **not** to indent here, isn't it?

Reply:
Yeah, perhaps you are right.  Could you please provide a patch with no indents for `#include` statements?

Reply:
Done